config/configuration.json

```diff
@@ -51,6 +51,12 @@
       "difficulty": "HARD"
     }
   },
+  "ApproximateWayCheck": {
+    "deviation.minimum.meters": 35.0,
+    "angle.minimum": 100.0,
+    "bezierStep": 0.01,
+    "highway.minimum": "service"
+  },
```

Comment:
Could you please add a `challenge` object to this configuration. You can look at other checks' configurations for examples. This information is used when creating MapRoulette challenges.